ionoscloud/resource_k8s_node_pool.go

```diff
@@ -468,6 +460,18 @@ func resourcek8sNodePoolCreate(ctx context.Context, d *schema.ResourceData, meta
 		k8sNodepool.Properties.MaintenanceWindow.DayOfTheWeek = &mdVal
 	}
 
+	if autoscaling, err := getAutoscalingData(d); err != nil {
+		return diag.FromErr(err)
+	} else {
```

Comment:
probably don't need else?

Reply:
i can use autoscaling just in the if-else statement since it's not seen outside of it

Reply:
i can use autoscaling just in the if-else statement since it's not seen outside of it